ODLA/include/ODLA/ops/odla_ops_math.h

```diff
@@ -41,6 +41,30 @@ extern "C" {
 extern ODLA_API_EXPORT odla_value ODLA_API_CALL
 odla_Abs(odla_value input, const odla_value_id value_id);
 
+//! \brief Check whether element of input is a number
+/*!
+  IsNaN returns the bool array \p input.
+
+  \param input the input value
+  \param value_id a unique value id (can be NULL)
+
+  \return odla_bool
+*/
+extern ODLA_API_EXPORT odla_value ODLA_API_CALL
+odla_IsNaN(odla_value input, const odla_value_id value_id);
+
+//! \brief Check infinity
+/*!
+  IsNaN returns the bool array \p input.
+
+  \param input the input value
+  \param value_id a unique value id (can be NULL)
+
+  \return odla_bool
+*/
+extern ODLA_API_EXPORT odla_value ODLA_API_CALL
+odla_IsInf(odla_value input, const odla_value_id value_id);
+
```

Comment:
Function names in header files should be sorted like  Abs, Add, ... 
Some function names are out of order. We should fix them later